### PR TITLE
Fix typo in tests

### PR DESCRIPTION
**Pull request by mikehaertl**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  |❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: tests/Unit/ContainerTest.php [hunk 1]
```diff
@@ -218,7 +218,7 @@ public function testDependencies(): void
     public function testCircularReference(): void
     {
         $container = new Container([
-            TreeItem::class, TreeItem::class,
+            TreeItem::class => TreeItem::class,
```
Comment (mikehaertl) [inline tests/Unit/ContainerTest.php:221]: @samdark Maybe there should be a check for `is_string()` in `setMultiple() to avoid these type of errors?

Comment (samdark) [reply tests/Unit/ContainerTest.php:221]: Yeah.

Comment (mikehaertl) [reply tests/Unit/ContainerTest.php:232]: Added a check.

**Comment by samdark** [comment]
Thanks!

**Pull request merged**